django/contrib/gis/measure.py

```diff
@@ -233,6 +233,7 @@ class Distance(MeasureBase):
         'clarke_link': 0.201166195164,
         'fathom': 1.8288,
         'ft': 0.3048,
+        'furlong': 201.168,
```

Comment:
Would it need an entry in `ALIAS`? The unit doesn't have a long-form alias like the others in that list. The only distinction would be upper or lower case, which is caught by [line 190 in `measure.py`](https://github.com/django/django/blob/master/django/contrib/gis/measure.py#L190).

Cf. `chain` is in `UNIT` but not `ALIAS` because it doesn't have an alternate usage name, while `um` is in `UNIT` and in `ALIAS` as `micrometer`/`micrometre` due to alternative spelling.

I can certainly add it for completeness if you'd like.

I will go ahead and add a test for the furlong. I can do add tests for the other units too as another PR, if you think that would help too.